frontend/pages/team.js

```diff
@@ -28,25 +48,20 @@ const Team = () => {
         <div className='row'>
           <div className='col'>
             <div className='container'>
-              <div className='row'>
-                <ul className={style.ul}>
-                  <div className='text-center'>
-                    <li className={style.li}>
-                      <input type='text' id='person-search-input' placeholder='Search...' />
-                    </li>
-                    <li className={style.li}>
-                      <i className='search-icon'>O</i>
-                    </li>
-                    <li className={`${style.li} ${style.filtermenu}`}>
-                      <select className='filter-select'>
-                        <option value='Default'>Filter</option>
-                        <option value='Class'>Class</option>
-                        <option value='Degree'>Degree</option>
-                        <option value='Group'>Group</option>
-                      </select>
-                    </li>
-                  </div>
-                </ul>
+              <div className='row justify-content-between'>
+                <div className='col col-sm-4'>
+                  <label htmlFor='team-search' className='form-label'>Search</label>
+                  <input type='search' className='form-control' id='team-search' onChange={searchHandler} placeholder='Search by name, class, degree, group' />
+                </div>
```

Comment:
If we're getting rid of / commenting out the following section, please remove col-sm-4.

Remember, Bootstrap is a base 12 system meaning that there are always 12 columns.  Make sure that you expand items out as necessary.  Meaning, if the following section has: 
`col col-sm-4 col-md-3 col-lg-2`, then this section should have `col col-sm-8 col-md-9 col-lg-10`